### PR TITLE
Correct the docstring of Head func in convnext.py

### PR DESCRIPTION
**Pull request by Frightera**
Small fixes on `def Head()` function's docstring as it was a copy from `RegNet`.

### CODE REVIEW

File: keras/applications/convnext.py [hunk 1]
```diff
@@ -328,15 +328,15 @@ def apply(x):
...
+    """Implementation of classification head of ConvNeXt.
 
     Args:
       num_classes: number of classes for Dense layer
       classifier_activation: activation function for the Dense layer
       name: name prefix
 
     Returns:
-      Classification head function.
+      Tensor of logits or softmax values as the output.
```
Comment (qlzh727) [inline keras/applications/convnext.py:339]: I think the existing return string is still correct, since it only returns a function that produce a logits tensor, but not the logits tensor itself.

**Review by qlzh727 — CHANGES_REQUESTED** [review]

File: keras/applications/convnext.py (hunk 1 above)
Comment (Frightera) [reply keras/applications/convnext.py:339]: Hmm I see, maybe I can also edit `regnet.py` to make them consistent, right now regnet head function has this:
https://github.com/keras-team/keras/blob/0ce152e06a8b467a4071ff3d1324406374f007e9/keras/applications/regnet.py#L837-L844

What do you think? @qlzh727

Comment (Frightera) [reply keras/applications/convnext.py]: I think `Classification head function` would make more sense.

**Review by qlzh727 — APPROVED** [review]
